web/sites/default/config_prod/mailsystem.settings.yml

```diff
@@ -2,12 +2,12 @@ _core:
   default_config_hash: IhwTepsVwtbtbcT5GzQKhCXDCRvbk3MNkWqPiuiZ10s
 theme: cloudy2
 defaults:
-  sender: sendgrid_integration
+  sender: test_mail_collector
```

Comment:
I just realized I think we decided we wanted to leave this set to sendgrid since the user sync is disabled now so that we'll continue to get the content emails.

I'll set it to sendgrid using the UI, but we'll need to fix it in the config in a future story.